poetry/utils/env.py

```diff
@@ -879,9 +879,13 @@ class Env(object):
 
     def __init__(self, path: Path, base: Optional[Path] = None) -> None:
         self._is_windows = sys.platform == "win32"
+        self._is_mingw = sysconfig.get_platform() == "mingw"
 
+        if not self._is_windows or self._is_mingw:
+            bin_dir = "bin"
+        else:
+            bin_dir = "Scripts"
```

Comment:
```suggestion
        bin = {"win32": "Scripts"}.get(sys.platform, "bin")
```

Would this be slightly better maintainable considering we only care about exceptions?

Reply:
This doesn't work:
```
>>> bin = {"win32": "Scripts"}.get(sys.platform, "bin")
>>> print(bin)
Scripts
```

Reply:
Given msys seems to only patch `get_platform()` , guess that is expected.